src/Import/Remote/PackageImporter.php

```diff
@@ -77,15 +88,14 @@ public function fetchPackages(): bool
         $forceUpdate = true !== $this->input->getParameterOption('--update-remotes', true);
         $displayProgress = 'list' !== $this->input->getFirstArgument();
 
-        $autoloadPath = PathHelper::getRoot() . Composer::VENDOR_DIR . 'autoload.php';
-
-        if (!file_exists($autoloadPath)) {
+        // FIXME: really needed?
```

Comment:
cc @pyrech

Reply:
It was one way to easily detect if somebody manually removed their .castor/vendor (in case of troubles or to force update). It may no longer be needed, i don't have the current implementation in mind